test/test.ts

```diff
@@ -1,13 +1,34 @@
 import { expect } from 'chai'
 import 'mocha'
-
-/*tslint:disable*/
-const fixtureJson = require('./fixture.json')
-const outputJson = require('./out/api_data.json')
-/*tslint:enable*/
-
+import * as apidoc from 'apidoc'
+import * as fs from 'fs-extra'
+import * as path from 'path'
+const tests = [
+  {
+    only: false,
+    name: 'test1'
+  },
+  {
+    skip: true,
```

Comment:
why skipping some tests?

Reply:
Because they are broken. But we keep them to know what does not work. (For example Array<MyClass>)